css_/team.css

```diff
@@ -67,19 +72,21 @@ span#toggle-icon
 {
     color: black;
 }
-.heading
+#heading
 {
-    margin-top: 53px;
```

Comment:
on changing the value of margin to 53px this is how the navbar looks :
![screenshot 2](https://user-images.githubusercontent.com/26070070/32356131-137742f8-c059-11e7-9d33-fb246c4a93ab.png)

It has the same scrolling effect of that of the home page:
![screenshot 3](https://user-images.githubusercontent.com/26070070/32356191-6e16e5c4-c059-11e7-965d-20886ac4c760.png)